build.gradle

```diff
@@ -7,7 +7,6 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.opencsv:opencsv:5.7.1'
```

Comment:
I don't think you should delete this. GitHub auto grading is showing a bunch of errors now because it cannot find the OpenCSV libraries, etc.